_layouts/container.html

```diff
@@ -33,6 +33,7 @@ <h1 class="page-title">{{ page.title }}</h1>
 <!-- Components -->
 <script src="{{ site.baseurl | append: '/media/js/presidium.js' | replace: '//','/'}}"></script>
 <script>
+    window.presidium['versions'] = { siteroot: "{{ site.siteroot }}" };
     window.presidium.menu.load({% include menu.json %}, 'presidium-navigation');
```

Comment:
We can move: siteRoot to the generated menu.json:
https://github.com/SPANDigital/presidium-core/blob/develop/src/menu.js#L36

no need to attach presidium['versions']

 
window.presidium.menu.load({% include menu.json %}, 'presidium-navigation', )